cmd/allocator/main.go

```diff
@@ -180,7 +222,9 @@ func (h *httpHandler) postOnly(in handler) handler {
 }
 
 type httpHandler struct {
-	agonesClient versioned.Interface
+	allocator          *gameserverallocations.Allocator
```

Comment:
I don't see that the allocator or stop fields are used anywhere. Why are they part of the struct instead of being defined locally in the `newServiceHandler` function?